test/go-tests/test_webhook.go

```diff
@@ -490,7 +490,7 @@ func getWebhookYamlWithSubscriptionIDs(t *testing.T, taskTypes []string, project
 	return webhookYamlWithSubscriptionIDs
 }
 
-func deleteFile(t *testing.T, shipyardFilePath string) {
+func DeleteFile(t *testing.T, shipyardFilePath string) {
```

Comment:
let's give the `shipyardFilePath` a more generic name here, as the function can also be used for deleting other files.